### ISSUE TITLE
Support mapping both measurements and their values

### ISSUE THREAD

**Issue by schuemie** [OHDSI/Usagi#17]
Currently, if someone wants to map measurements and their values (in case the measurement has a defined set of values like 'positive' and 'negative') they will need to map those separately. It might make sense to link a measurement to its possible values.

**Comment by anonymous34** [comment]
@MaximMoinat 

Yes this looks like the right direction.  We also have other pairs like Family History and medical history in the observation table where some examples look like this:

![image](https://user-images.githubusercontent.com/33704272/106532142-3f5f4b00-64b5-11eb-988a-1c2ff170894e.png)

Is this enhancement being added soon and when can it be tested?
